smac/epm/random_epm.py

```diff
@@ -20,17 +20,10 @@ class RandomEPM(AbstractEPM):
     rng : np.random.RandomState
```

Comment:
Not sure whether we use this anymore, but shouldn't this switch to seed as well?

Reply:
I removed the comment as it should be inherited in the documentation.